firebase-authexchange/firebase-authexchange.gradle.kts

```diff
@@ -0,0 +1,53 @@
+plugins {
```

Comment:
Good catch! I had a script I wrote that would automatically add it to any missing files, but it doesn't auto-run. We should probably have some sort of check regarding that. Not anything that's enforced, since there are way too many exceptions, but maybe like a git hook or gha that looks at files added and lets you know if any of interest are missing copyright notices (`.kt`,`.java`,etc.,).

Reply:
LOL, well I'll be paying more attention to that one for sure